card/lib/card/migration.rb

```diff
@@ -29,11 +29,11 @@ def find_unused_name base_name
     end
 
     def paths mig_type=type
-      Cardio.paths["db/migrate#{schema_suffix mig_type}"].to_a
+      Card.paths["db/migrate#{schema_suffix type}"].to_a
```

Comment:
our standard way to test migrations is to run all migrations from 1.10.  See data dump in wagn/test.

Migration debugging process is roughly:
1. import sql to db
2. check out 1.10
3. config to use db
4. run `wagn update`
5. fix bugs
6. repeat